libpay-stripe/src/main/scala/com/wix/pay/stripe/ChargeMetadataToEmailHelper.scala

```diff
@@ -0,0 +1,87 @@
+package com.wix.pay.stripe
+
+import com.wix.pay.creditcard.CreditCard
+import com.wix.pay.model.{Customer, Deal, OrderItem}
+import com.wix.pay.stripe.ChargeMetadataToEmailHelper._
+
+import scala.collection.immutable.ListMap
+
+class ChargeMetadataToEmailHelper {
```

Comment:
Why is this class name contains word "email"? Stripe documentation says: "Any metadata you include is viewable in the Dashboard (e.g., when looking at the page for an individual charge), and is also available in common reports and exports." I think we should remove word "Email" from this method name.